tracer/src/Datadog.Trace/Ci/Agent/CIAgentWriter.cs

```diff
@@ -75,7 +91,15 @@ public void WriteTrace(ArraySegment<Span> trace)
                 finalTrace[idx++] = span;
             }
 
-            _agentWriter.WriteTrace(new ArraySegment<Span>(finalTrace, 0, idx));
+            if (spanIdsToRemove != null)
+            {
+                CIVisibility.Log.Warning($"Spans dropped because not having a test or benchmark root span: {spanIdsToRemove.Count}");
```

Comment:
```suggestion
                CIVisibility.Log.Warning("Spans dropped because not having a test or benchmark root span: {Count}", spanIdsToRemove.Count);
```

Reply:
Didn't use that because there's a Warning overload with the line number and column, forcing me to create an array instead.
![image](https://user-images.githubusercontent.com/69803/135474054-5f54b648-d54f-487d-87a9-96ce2784d5e0.png)

Reply:
Good point, this is the one you want: 
```suggestion
                CIVisibility.Log.Warning<int>("Spans dropped because not having a test or benchmark root span: {Count}", spanIdsToRemove.Count);
```

Reply:
Oh great!.